apps/percona/chaos/node_freeze/test.yml

```diff
@@ -61,20 +55,29 @@
           include_tasks: "/common/utils/mysql_data_persistence.yml"
           vars:
             status: 'LOAD'
-            ns: "{{ namespace }}"
+            ns: "{{ app_ns }}"
             pod_name: "{{ app_pod_name.stdout }}"
```

Comment:
@vibhor check whether it is {{ namespace }} or {{ app_ns }} ???

Reply:
@nsathyaseelan done